python/semantic_kernel/connectors/openapi_plugin/openapi_manager.py

```diff
@@ -29,6 +31,18 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
+@experimental_class
+class OperationExtensions(Enum):
+    """The operation extensions."""
+
+    METHOD_KEY = "method"
+    OPERATION_KEY = "operation"
+    INFO_KEY = "info"
+    SECURITY_KEY = "security"
+    SERVER_URLS_KEY = "server-urls"
+    METADATA_KEY = "operation-extensions"
```

Comment:
Just making sure: the value of `METADATA_KEY` is not `metadata`. And I am not seeing it being used.

Reply:
Good catch. I need to implement OpenAPI extensions, but I don't think it'll come in this PR.